python/ray/util/tracing/tracing_helper.py

```diff
@@ -340,6 +340,11 @@ def _inject_tracing_into_function(function):
         ),
     )
 
+    # Skip wrapping if tracing is disabled (still add _ray_trace_ctx however to make
+    # sure _ray_trace_ctx could be passed)
+    if not _is_tracing_enabled():
```

Comment:
Have to change this b/c local module loading is broken right now b/c of this wrapper (regardless of whether tracing is enabled or not): we'd try to load this module `tracing_helper` that isn't available in the local runtime environment

cc @architkulkarni @scv119

Reply:
seems fine to me.

Reply:
Seems like a pretty big issue (though not sure how prevalent is local code loading among Ray users)

I've added a test easily reproducing this and created https://github.com/ray-project/ray/issues/38633